airmozilla/main/templates/main/home.html

```diff
@@ -11,7 +11,17 @@
 </style>
 {% endblock %}
 
-
+{% block extra_head %}
+{{ super() }}
+<meta name="keywords" content="{{ ','.join(tags) }}">
```

Comment:
Yes you are right, It does not have an og property so it is a normal meta-tag, I'll remove that.